FreeRTOS-Plus/Source/Application-Protocols/network_transport/mbedtls_pk_pkcs11.c

```diff
@@ -1020,7 +1137,12 @@ static CK_RV p11_rsa_ctx_init( void * pvCtx,
         xResult = CKR_FUNCTION_FAILED;
     }
 
-    /* / * Initialize public RSA parameter data from attributes * / */
+    /*
+     * TODO: corePKCS11 does not allow exporting RSA public attributes.
+     * This function should be updated to properly initialize the
+     * mbedtls_rsa_context when this addressed.
```

Comment:
```suggestion
     * mbedtls_rsa_context when this is addressed.
```

Reply:
Is this clearly documented anywhere other than just this comment?

Reply:
Nope. RSA is really not well supported by corePKCS11 other than basic signature operations.